### PR TITLE
feat(ui): change "VM host" to "KVM host"

### PR DESCRIPTION
**Pull request by huwshimi**
## Done

- Update the host labels from "VM host" to "KVM host".

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- This could probably just have a code review, but you could have a poke around the LXD area to make sure everything says "KVM host" instead of "VM host".

## Fixes

Fixes: canonical-web-and-design/app-tribe#506.

### CODE REVIEW

File: ui/src/app/kvm/components/KVMHeaderForms/AddLxd/AddLxd.test.tsx [hunk 1]
```diff
@@ -86,7 +86,7 @@ describe("AddLxd", () => {
   });
 
   it(`shows the authentication form if the user has generated a certificate for
-    the LXD VM host`, () => {
+    the LXD KVM host`, () => {
```
Comment (huwshimi) [inline ui/src/app/kvm/components/KVMHeaderForms/AddLxd/AddLxd.test.tsx:89]: I updated the test names and comments as well so we're using the same language everywhere.

File: ui/src/app/kvm/components/LXDVMsTable/VMsTable/VMsTable.tsx [hunk 2]
```diff
@@ -260,7 +262,8 @@ const VMsTable = ({
       {searchFilter && vms.length === 0 ? (
         <Strip shallow rowClassName="u-align--center">
           <span data-test="no-vms">
-            No VMs in this VM host match the search criteria.
+            No VMs in this {displayForCluster ? "cluster" : "KVM host"} match
```
Comment (huwshimi) [inline ui/src/app/kvm/components/LXDVMsTable/VMsTable/VMsTable.tsx:265]: Drive-by fix for the empty search  message.

File: ui/src/app/kvm/views/KVMList/LxdTable/LxdKVMHostTable/LxdKVMHostTable.tsx [hunk 3]
```diff
@@ -101,7 +102,9 @@ const generateRows = (rows: LxdKVMHostTableRow[]) =>
               }
               secondary={
                 isCluster ? (
-                  <span data-test="hosts-count">{row.hostsCount} VM hosts</span>
+                  <span data-test="hosts-count">
+                    {pluralize("KVM host", row.hostsCount, true)}
```
Comment (huwshimi) [inline ui/src/app/kvm/views/KVMList/LxdTable/LxdKVMHostTable/LxdKVMHostTable.tsx:106]: Drive-by fix to pluralise the label.

**Review by Caleb-Ellis — CHANGES_REQUESTED** [review]
A few other places we could be consistent:
- Refresh KVM form. We should change the form text and submit button to say "KVM host" instead of just "KVM".
- Delete KVM form. Same as above.
- Add LXD form. Not directly related, but in the last step in the form when you actually add the LXD host the submit button just says "Next". The behaviour was changed to no longer redirect to the newly added LXD details page, so it should probably say "Save LXD host" instead.
- Add Virsh form. The submit button could be changed from "Save KVM" to "Save Virsh host".

**Comment by huwshimi** [comment]
@Caleb-Ellis ready for another review.

File: ui/src/app/kvm/components/KVMHeaderForms/RefreshForm/RefreshForm.tsx [hunk 4]
```diff
@@ -29,7 +30,7 @@ const RefreshForm = ({
       clearHeaderContent={clearHeaderContent}
       errors={errors}
       initialValues={{}}
-      modelName="KVM"
+      modelName={pluralize("KVM host", hostIds.length)}
```
Comment (Caleb-Ellis) [inline ui/src/app/kvm/components/KVMHeaderForms/RefreshForm/RefreshForm.tsx:33]: This should just be "KVM host". `ActionForm` already handles pluralising, so this results in "KVM hostss"

**Review by Caleb-Ellis — APPROVED** [review]
Just one little comment

File: ui/src/app/kvm/components/KVMHeaderForms/RefreshForm/RefreshForm.tsx (hunk 4 above)
Comment (huwshimi) [reply ui/src/app/kvm/components/KVMHeaderForms/RefreshForm/RefreshForm.tsx]: Thanks!
